src/api/files.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const argCommand = require('../cmd-helpers').argCommand
+
+module.exports = send => {
+  return {
+    cp: argCommand(send, 'files/cp'),
+    ls: argCommand(send, 'files/ls'),
+    mkdir: argCommand(send, 'files/mkdir'),
+    stat: argCommand(send, 'files/stat'),
+    rm: function (path, opts, cb) {
```

Comment:
`files.rm` is defined in such a way that requires `opts` to be passed. When recursively deleting is not necessary and `opts` is omitted, the function doesn't work, it makes an http request for rm without the `path` argument, so it's mandatory to pass an empty `{}` as opts.

Now that I think of it, how can you _not_ delete recursively? :confused: